step.yml

```diff
@@ -115,10 +115,22 @@ inputs:
         - "no"
   - env_var_lookup_keys:
     opts:
-      title: "List of env var keys to lookup"
+      title: "List of env var keys for lookup to determine cache fingerprint (Deprecated, use `cache_env_var_lookup_keys` instead)"
       summary: "A list of env variable keys to lookup in order to determine whether the build should be cached or not."
       description: "A list of env variable keys to lookup in order to determine whether the build should be cached or not."
       is_expand: false
+  - cache_env_var_lookup_keys:
+    opts:
+      title: "List of env var keys for lookup to determine cache fingerprint (Deprecated, use `cache_env_var_lookup_keys` instead)"
+      summary: "A list of env variable keys to lookup in order to determine whether the build should be cached or not."
+      description: "A list of env variable keys to lookup in order to determine whether the build should be cached or not."
+      is_expand: false
+  - cache_file_lookup_paths:
+    opts:
+      title: "List of files for lookup to determine cache fingerprint (Deprecated, use `cache_env_var_lookup_keys` instead)"
```

Comment:
Here we should remove the "deprecated" message

Reply:
😱 good catch!